js/jquery.mapael.js

```diff
@@ -1293,32 +1276,18 @@
 
                 textPosition = self.getTextPosition(bbox, elemOptions.text.position, elemOptions.text.margin);
                 if (textPosition.x !== elem.textElem.attrs.x || textPosition.y !== elem.textElem.attrs.y) {
-                    if (animDuration > 0) {
-                        elem.textElem.attr({"text-anchor": textPosition.textAnchor});
-                        elem.textElem.animate({x: textPosition.x, y: textPosition.y}, animDuration);
-                    } else
-                        elem.textElem.attr({
-                            x: textPosition.x,
-                            y: textPosition.y,
-                            "text-anchor": textPosition.textAnchor
-                        });
+                    self.animate(elem.textElem, {x: textPosition.x, y: textPosition.y}, animDuration);
```

Comment:
What about the 'text-anchor' attribute ?

Reply:
Ouh right. This one got squashed in the process! I will fix this ASAP :)